pkg/cluster/kubernetes/resource/load.go

```diff
@@ -8,14 +8,29 @@ import (
 	"os"
 	"path/filepath"
 
+	sops "go.mozilla.org/sops/v3"
+	"go.mozilla.org/sops/v3/decrypt"
 	"github.com/pkg/errors"
 	"gopkg.in/yaml.v2"
 )
 
+// decryptionStrategy is a type of function that reads data and return the decrypted data
+type decryptionStrategy func([]byte) ([]byte, error)
```

Comment:
Would you mind adding that this is needed for testing? (mocking the strategy)

Reply:
This is not needed anymore, right?

Reply:
Correct. There is no need to mock up the decryption strategy any more